packages/appcache/appcache-server.js

```diff
@@ -39,6 +39,13 @@ var browserDisabled = function (request) {
   return disabledBrowsers[request.browser.name];
 };
 
+function isDynamic (resource) {
```

Comment:
Since this is a named function, please remove the space between the function name and the parens: `function isDynamic(resource) {`.